docs/getting-started/summaryofchanges.md

```diff
@@ -76,6 +81,14 @@ The following bugs were fixed.
 
 #### Zowe App Server
 
+- Bugfix: ZSS will now maintain the connection if users respond to the 404 message with the request `Connection: Keep-Alive` [#147](https://github.com/zowe/zowe-common-c/pull/147)
+  - **NOTE:** The code only recognizes `Connection: Keep-Alive`.  Other "Keep-Alive" properties will be ignored.
+- Bugfix: If a loadmodule is incorrectly copied to STEPLIB, the z/OS loader will fail to load it. In these cases, an available copy in LPA will be used instead, if one is available. The problem with LPA is that any IDENTIFY calls to a module with an incorrect version number may cause serious issues. This pull-request ensures that ZWESIS01 comes from private storage. [#146](https://github.com/zowe/zowe-common-c/pull/146)
+- Bugfix: Fixes various issues that would occur when the number in the `Content-length` response header was different from the actual content length. [#150](https://github.com/zowe/zowe-common-c/pull/150)
+- Bugfixes for default plugin config and terminal handler location [#229](https://github.com/zowe/zlux-server-framework/pull/229)
+  - This fix allows the serverside plugin config to exist within its own folder, rather than in the instance directory. As a result, plugins no longer have to perform a copy operation during installation.   
```

Comment:
For more info, see the wiki https://github.com/zowe/zlux/wiki/Configuration-Dataservice#packaging-defaults
This change was for including `_internal`. `storageDefaults` other than `_internal` were already previously supported.